### PR TITLE
Upgrade to Quarkus 2.6.3

### PR DESCRIPTION
**Pull request by sophokles73**
This is for #3056 

Explicitly override Kafka client version to keep using 2.8.1 because
3.0.0 (as managed by Quarkus) is not compatible with vertx-kafka 4.2.2
(yet).

### CODE REVIEW

**Comment by sophokles73** [comment]
@calohmn would you mind taking a look?

**Review by calohmn — APPROVED** [review]
LGTM

File: bom/pom.xml [hunk 1]
```diff
@@ -45,27 +45,26 @@
...
-    <k8s-client.version>5.10.2</k8s-client.version>
-    <log4j-api.version>2.17.1</log4j-api.version>
     <logback.version>1.2.9</logback.version>
-    <micrometer.version>1.7.6</micrometer.version>
     <mongodb-image.name>mongo:4.4</mongodb-image.name>
-    <mongodb-driver.version>4.3.4</mongodb-driver.version>
     <native.image.name>registry.access.redhat.com/ubi8/ubi-minimal:8.4</native.image.name>
     <netty.version>4.1.72.Final</netty.version>
     <netty.tcnative.version>2.0.46.Final</netty.tcnative.version>
-    <opentracing.version>0.33.0</opentracing.version>
```
Comment (calohmn) [inline bom/pom.xml:60]: @sophokles73 Only noticed this now:
The `opentracing.version` property is still referenced in the hono-core pom in the `maven-bundle-plugin` definition.
Without the property defined, a Hono build with the `embed-dependencies` maven profile fails.

Comment (sophokles73) [reply bom/pom.xml:60]: I guess we (that is: you) can create an issue for it. However, I doubt that anybody is (still) trying to deploy the client module as an OSGi bundle.

**Pull request merged**